contrib/FreeBSD/README

```diff
@@ -0,0 +1,46 @@
+This folder contains sopel and sopel-default.cfg designed to be distributed by third parties such as FreeBSD.
+
+`sopel-default.cfg` is a default configuration file for sopel. The -default prefix is mandatory and will be used as a 'profile'. The profiles may be changed with the variable `sopel_profiles` in `/etc/rc.conf`. For example:
+
+```sh
```

Comment:
If using markdown formatting, this file should be named *.md

Reply:
I will change the extension